assets/dev/js/frontend/utils/anchor-scroll-margin.js

```diff
@@ -0,0 +1,160 @@
+module.exports = elementorModules.ViewModule.extend( {
+	getDefaultSettings() {
```

Comment:
Just a question, this is a lot of JS, how did we "reduce the use of JS"? :)

Reply:
@louiswol94 If I understand it correctly, the code will only run once, when there are no sticky elements. Then it will only assign the scroll margin values on page load.

When there are sticky elements with a scroll margin top setting on the page, then this code will run more often. But only then. I think that is acceptable.

The scrolling itself is not handled by JS, which causes some of the Accessibility issues.